mayastor/src/replicas/rebuild/rebuild_impl.rs

```diff
@@ -585,3 +462,175 @@ impl RebuildJob {
         }
     }
 }
+
+#[derive(Debug)]
+pub(super) struct RebuildStates {
+    /// Current state of the rebuild job
+    pub current: RebuildState,
+
+    /// Pending state for the rebuild job
+    pending: Option<RebuildState>,
+}
+
+impl std::fmt::Display for RebuildStates {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        write!(f, "{:?}", self)
+    }
+}
+
+impl Default for RebuildStates {
+    fn default() -> Self {
+        Self {
+            current: RebuildState::Init,
+            pending: None,
+        }
+    }
+}
```

Comment:
Given that these are being set to their default values, could we use Default::default()?

Reply:
Done